tests/validations/test_unique_valid_name.py

```diff
@@ -19,11 +19,11 @@
     Top level elements include
     - Semantic Models
     - Metrics
-    - Metric Sets
+    - PydanticMetric Sets
     - Dimension Sets
 
     A name for any of these elements must be unique to all other top level elements
-    except metrics. Metric names only need to be unique in comparison to other metric
+    except metrics. PydanticMetric names only need to be unique in comparison to other metric
```

Comment:
Neither of these need to be changed, the Metric here is the generic construct rather than the implementation class.

Reply:
Let me fix this at the end of the stack.